v2/driver/driver.go

```diff
@@ -116,16 +111,17 @@ func RootDAG(ctx context.Context, opts Options, mlmd *metadata.Client) (executio
 	if err != nil {
 		return nil, err
 	}
-	ecfg.IsRootDAG = true
 	ecfg.ExecutionType = metadata.DagExecutionTypeName
+	ecfg.PipelineRoot = pipeline.GetPipelineRoot()
+	ecfg.Name = fmt.Sprintf("run/%s", opts.RunID)
```

Comment:
Note, root execution will have name: "run/<run-id>", so that we can easily find root execution of a pipeline run.